strawberry/field.py

```diff
@@ -187,7 +202,11 @@ def is_basic_field(self) -> bool:
         an `Info` object and running any permission checks in the resolver
         which improves performance.
         """
-        return not self.base_resolver and not self.permission_classes
+        return (
+            not self.base_resolver
+            and not self.permission_classes
```

Comment:
Since we are converting permissions to extensions now, we don't need to check for this anymore. Just base_resolve and extensions

```suggestion
```